packages/kbn-config-schema/src/types/object_type.test.ts

```diff
@@ -30,13 +30,20 @@ test('returns value by default', () => {
   expect(type.validate(value)).toEqual({ name: 'test' });
 });
 
+test('returns empty object if undefined', () => {
+  const type = schema.object({});
+  expect(type.validate(undefined)).toEqual({});
+});
+
 test('fails if missing required value', () => {
   const type = schema.object({
     name: schema.string(),
   });
   const value = {};
 
-  expect(() => type.validate(value)).toThrowErrorMatchingSnapshot();
+  expect(() => type.validate(value)).toThrowErrorMatchingInlineSnapshot(
```

Comment:
I made all these tests use inline snapshots so I could read them here. None of the existing assertions should have changed.